TAF_EasyRest/TestFramework/PageComponents/AdministratorPanelComponents/WaitingForConfirmListPageComponent.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class WaitingForConfirmListPageComponent
+    {
+        IWebDriver driver;
```

Comment:
please specify the access modifier explicitly

Reply:
added